Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+flake8 = "*"
+setuptools = ">=30.3.0"
+setuptools-scm = "*"
+wheel = "*"
+autopep8 = "*"
+nose = "*"
+importlib-metadata = "*"
+
+[packages]
+launchable = {editable = true,path = "."}
+
+[scripts]
+build = "python setup.py sdist bdist_wheel"
+install = "pip install -U ."
+test = "python -m nose tests"
+lint = "flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics"
+lint-warn = "flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics"
```

Comment:
How did you decide these values?

Reply:
I copied these rules from a [GithubActions template](https://github.com/actions/starter-workflows/blob/main/ci/python-package.yml). We already have been using it on our Github Actions. I think we should consider what rule we should use later.